src/kiwixapp.cpp

```diff
@@ -307,6 +307,36 @@ void KiwixApp::createAction()
     CREATE_ACTION(NewTabAction, gt("new-tab"));
     SET_SHORTCUT(NewTabAction, QKeySequence::AddTab);
 
+    CREATE_ACTION(ToggleTab1Action, gt("toggle-tab-1"));
+    SET_SHORTCUT(ToggleTab1Action, QKeySequence(Qt::ALT + Qt::Key_1));
+
+    CREATE_ACTION(ToggleTab2Action, gt("toggle-tab-2"));
+    SET_SHORTCUT(ToggleTab2Action, QKeySequence(Qt::ALT + Qt::Key_2));
+
+    CREATE_ACTION(ToggleTab3Action, gt("toggle-tab-3"));
+    SET_SHORTCUT(ToggleTab3Action, QKeySequence(Qt::ALT + Qt::Key_3));
+
+    CREATE_ACTION(ToggleTab4Action, gt("toggle-tab-4"));
+    SET_SHORTCUT(ToggleTab4Action, QKeySequence(Qt::ALT + Qt::Key_4));
+
+    CREATE_ACTION(ToggleTab5Action, gt("toggle-tab-5"));
+    SET_SHORTCUT(ToggleTab5Action, QKeySequence(Qt::ALT + Qt::Key_5));
+
+    CREATE_ACTION(ToggleTab6Action, gt("toggle-tab-6"));
+    SET_SHORTCUT(ToggleTab6Action, QKeySequence(Qt::ALT + Qt::Key_6));
+
+    CREATE_ACTION(ToggleTab7Action, gt("toggle-tab-7"));
+    SET_SHORTCUT(ToggleTab7Action, QKeySequence(Qt::ALT + Qt::Key_7));
```

Comment:
Maybe Qt::AltModifier | Qt::Key_7  is a better option, according to
 https://doc.qt.io/qt-5/qt.html#KeyboardModifier-enum